src/SwedbankPay.Sdk/PaymentInstruments/PaymentAbortRequestData.cs

```diff
@@ -0,0 +1,15 @@
+﻿namespace SwedbankPay.Sdk.PaymentInstruments
+{
+    public class PaymentAbortRequestData
+    {
+        public PaymentAbortRequestData()
+        {
+            Operation = "Abort";
+            AbortReason = "CancelledByConsumer";
```

Comment:
This is a good pattern for request classes. Initialize default values, have public setters and allow the consumer to "build" the request class incrementally. Just be sure to validate the request object thoroughly before it is mapped to a DTO and a n HTTP request is performed.